DEVELOPER_GUIDE.md

```diff
@@ -24,6 +24,13 @@ This document will walk you through on what's needed to start contributing code
   - **Pyenv** : Install `pyenv` and follow the instructions in the output of `pyenv init` to set up your shell and restart it before proceeding.
     For more details please refer to the [PyEnv installation instructions](https://github.com/pyenv/pyenv#installation).
 
+    Install the following modules to continue with the next steps: 
```

Comment:
Is this specific to testing on a debian-based distro? If so, maybe we should include this at the end as optional based on OS system.

Reply:
Got it. Will add!